test/unit/foreman/access_permissions_test.rb

```diff
@@ -10,59 +11,8 @@
 #
 # In particular, it's important that actions for AJAX requests are added to
 # an appropriate permission so views using those requests function.
-class AccessPermissionsTest < ActiveSupport::TestCase
-  MAY_SKIP_REQUIRE_LOGIN = [
```

Comment:
http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Skipping-tests is provided if you want to exclude certain routes being tested, exclude the test.

Moving the constants will still cause the tests to be run from this file against the existing constants, it won't allow a plugin author to add additional constants. (The tests defined by this file will continue to fail.)